python/cudf/cudf/tests/test_query.py

```diff
@@ -209,3 +209,25 @@ def test_query_with_index_keyword(query, a_val, b_val, c_val):
     expect = pdf.query(query)
 
     assert_eq(out, expect)
+
+
+@pytest.mark.parametrize(
+    "data, query",
+    [
+        # Only need to test the
+        (["a", "b", "c"], "data == 'a'"),
+    ],
+)
+def test_query_unsupported_dtypes(data, query):
+    gdf = cudf.DataFrame({"data": data})
+
+    # make sure the query works in pandas
```

Comment:
Is this important? I'm fine doing so, it just seems a bit strange to be testing here and I'm not sure what it achieves other than basically functioning as a unit test for pandas. Do you want to verify that this behavior continues to work in pandas?

Reply:
my thinking here was that the test was meant to express the gap between the dtypes that pandas supports but we do not. To that end, I thought it might make it more complete to make sure the dtype being tested actually works in pandas at all.